lib/vue/src/components/map/CMap.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+<l-map
+  ref="map"
+  :zoom="map.zoom"
+  :center="map.center"
+  :min-zoom="map.zoomMin"
+  :max-zoom="map.zoomMax"
+  :bounds="map.bounds"
+  :max-bounds="map.maxBounds"
+  style="height: 75vh; width: 100%; cursor: pointer;"
+  :style="styles"
+  @click="placeMarker"
```

Comment:
This should be onMapClick, the actual marker placing needs to be handled where the c-map is used
The CMap only shows stuff, it shouldn't manipulate the markers/value